docs/api/index.rst

```diff
@@ -11,12 +11,18 @@ API
 Changelog
 ---------
 
+1.11 (unreleased)
+'''''''''''''''''
+
+- Parent metadata is now readable if children creation is allowed
+- Now returns a ``412`` instead of a ``403`` if the ``If-None-Match: *`` header is
+  provided and the ``create`` permission is allowed
```

Comment:
Sure, the tests and the code are there, but how was wondering if you wanted to add a note to the changelog about that?